gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
There is a wrapper configuration in the build.gradle file. Do you mind updating that and than calling wrapper task? That way everything would be consistent. Thanks.